hack/sast.sh

```diff
@@ -6,17 +6,24 @@
 
 set -e
 
-root_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." &> /dev/null && pwd )"
+report_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." &> /dev/null && pwd )"
 
 gosec_report="false"
 gosec_report_parse_flags=""
+exclude_dirs="hack"
 
 parse_flags() {
   while test $# -gt 1; do
     case "$1" in
       --gosec-report)
         shift; gosec_report="$1"
         ;;
+      --report-dir)
+        shift; report_dir="$1"
+        ;;
+      --exclude-dirs)
```

Comment:
Nit: what about naming the argument `--extra-exclude-dirs`?  Since we append w/e we provide here to the `hack` directory?

Reply:
Isn't it better to overwrite instead of to append? Some users of the script may want to check also the `hack` dir.